src/IdentityServer/src/Store.Mongo/Configuration/PersonalAccessTokenCollectionConfiguration.cs

```diff
@@ -37,7 +37,12 @@ private static void ConfigureCollection(IMongoCollection<IdOpsPersonalAccessToke
                 new(IndexKeys.Combine(IndexKeys.Ascending(WellKnownPatFields.IsUsed),
                         IndexKeys.Ascending(WellKnownPatFields.ExpiresAt),
                         IndexKeys.Ascending(x => x.UserName)),
-                    new CreateIndexOptions { Unique = true, Name = "PAT-Token-Search" })
+                    new CreateIndexOptions
```

Comment:
Does this explode when we deploy it?

Reply:
@PascalSenn:  💣 ---> 💥
No not really, as the index contains the "expiresAt" field we should be save as long as the pats do not expire at the exact same moment. But I dont really understand why the index is marked as unique. I will check with @philbir / @glucaci before merging.

Reply:
didn see the `_i` suffix :D